cmd/acb/commands/getsecret/getsecret.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package getsecret
+
+import (
+	"errors"
+	"log"
+
+	"github.com/Azure/acr-builder/vaults"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/urfave/cli"
+)
+
+// Command renders templates and verifies their output.
+var Command = cli.Command{
+	Name:  "getsecret",
```

Comment:
how will you add additional providers to this? maybe consider renaming it or making subgroups?
```
acb decrypt akv --fooFlag
acb decrypt foo --barFlag
```

Reply:
@ehotinger  The word decrypt sounds related to some form of decoding the original secret value. But here the secret is not decoded instead it is stored in a vault and the parameters help to locate them. I am thinking we will have the similar issue with the secrets object in task YAML file. How do we extend to support multiple vault providers? Is it something like 
```
secrets
  -id: akvsecret
    akv: "https://myvault.vault.azure.net/mysecret
  -id: awssecret
    awsv: "<don't know>"
```
So essentially  secrets will have all supported vaults properties and we will resolve them accordingly? If so getsecret can also follow the same pattern, It takes all properties supported by secrets as parameters and resolve them.